### ISSUE TITLE
Cannot install packages in Atom

### ISSUE THREAD

**Issue by ZappingStrike** [atom/atom#14670]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

Cannot install any packages, themes however install normally. This is the error I get:

```
assert.js:85
  throw new assert.AssertionError({
  ^
AssertionError: Expected there to only be one child in node_modules
    at C:\Users\Mark Mitchell\AppData\Local\atom\app-1.17.2\resources\app\apm\lib\install.js:174:22
    at ChildProcess.onChildExit (C:\Users\Mark Mitchell\AppData\Local\atom\app-1.17.2\resources\app\apm\lib\command.js:51:49)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:877:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5)
```

I have tried on beta, and on stable none worked. 

### Steps to Reproduce

-Just try to install any package, and it will error. Doesn't matter if it's in safe mode or not.

**Expected behavior:**  It will install the desired package.

**Actual behavior:** It does nothing and gives an error.

**Reproduces how often:** For me at least, it's 100%.

### Versions

Atom    : 1.17.2
Electron: 1.3.15
Chrome  : 52.0.2743.82
Node    : 6.5.0

apm  1.17.0
npm  3.10.5
node 6.9.5 x64
python
git 2.13.0.windows.1
visual studio 2013

Windows 10,  10.0.10586 N/A Build 10586

**Comment by rsese** [comment]
Just to confirm, did installing packages work fine in previous versions but started throwing this error for you as of 1.17.0 and 1.18.0-beta?

**Comment by ZappingStrike** [comment]
@rsese Yes, I do believe so. It worked before and then it stopped, I guess it updated automatically then.

**Comment by rsese** [comment]
Thanks for confirming - can you try temporarily resetting to factory defaults and see if you still get the same error?

http://flight-manual.atom.io/hacking-atom/sections/debugging/#reset-to-factory-defaults

**Comment by ZappingStrike** [comment]
I tried while reinstalling, I deleted the folder totally. I just tried again, unfortunately I got the same error again.

**Comment by 50Wliu** [comment]
And what if you attempt to install from the command line?  Syntax is `apm install package-name`.

**Comment by ZappingStrike** [comment]
Also gives the exact same error, I've already tried and just did again.

**Comment by damieng** [comment]
Not able to repro this on my machine.  I think if this was happening to everybody we'd be seeing a lot more reports. Not sure what could be causing that.

**Comment by ZappingStrike** [comment]
I can understand, but Atom is such a great programm, I would like to install packages too :D 
Any ideas how I could fix that then?

**Comment by damieng** [comment]
The problem is we have no idea what could be causing which would be the first part of figuring out a fix. You could perhaps try unpacking the atom-windows.zip portable version.

**Comment by damieng** [comment]
No response.

**Comment by JakeAndrew** [comment]
I could not install packages in Atom, either. Nobody else's tips worked for me, so below, I am posting what I did to get it to work.

I am running Fedora. Initially, I downloaded the Atom rpm package from the website, atom.io. I tried to install it by double-clicking it, but apper gave an error at the end.

I resort to Copr. There is a mosquito/atom package there. I enabled the repo and installed. At this point in the story, Atom is installed and working great.

The point of all this was just to edit a ReadMe with Markdown. I had the plain text on the left and the Markdown preview on the right, working spectacularly. Then something happens, an auto-update, I assume. After "the happening", whenever I edited the plain text, the preview would scroll to the bottom -- extremely counter-productive and very annoying. I find the markdown-sync-scroll package, which was created to remedy this exact problem. I try to install the package, and it fails. I get an error about a version being undefined. The last line said "not ok".

I try installing the package from terminal using apm, with and without sudo, same error. I try updating nodejs and doing "npm rebuild". Same error. Uninstalling, reinstalling atom -- same error.

Finally, I `sudo dnf remove --purge atom`. Then I delete `/etc/yum.repos.d/_copr_mosquito-atom.repo`. Then I go to the atom/atom ReadMe on github. I follow its link, download, and install *that* rpm by double-clicking it. This is *not* the one from the website, atom.io. Again, apper fails. But by luck, somehow I notice that it actually installed anyhow. I purged with dnf and tried to install the package via double-click several times before noticing that it actually does install, despite the apper error.

And this time, packages install. Now, I have Markdown editing how I want it! Yay!
